src/generic/gmsh_read.cc

```diff
@@ -0,0 +1,1355 @@
+//
+// Created by iqraa on 20-1-24.
```

Comment:
is this third party (library?) code? If so we should move it elsewhere. If it's ours it needs the oomph-lib standard boilerplate stuff.